pyproject.toml

```diff
@@ -17,11 +17,13 @@ certifi = ">=2023.7.22"
 frozendict = "^2.3.2"
 pillow = "^9.0.0" # TODO: We may want to mark pillow (and numpy) as extra (https://python-poetry.org/docs/master/pyproject#extras)
 pydantic = "^1.7.4"
-python = ">=3.7.0,<4.0"
+python = ">=3.9.0,<4.0"
```

Comment:
sphinx requires at least python 3.9 that's why.

Reply:
Some customers might still be using python 3.8 (it's not end-of-life yet - https://endoflife.date/python). So I'd recommend seeing if you can use a looser lower limit on the sphinx dependency that allows us to use `python >=3.8.0,<4.0`

Reply:
This is not good.  Canceling support for py3.7 and py3.8 is a big deal.  py3.7 is pretty old, but py3.8 is still default on some popular systems.

We should figure out how to build the docs on a newer python without cutting off SDK support.